app/controllers/reports/regions_controller.rb

```diff
@@ -39,15 +39,15 @@ def show
       slug = region.slug
       {
         region: region,
-        adjusted_patient_counts: repo.adjusted_patient_counts[slug],
-        controlled_patients: repo.controlled_patients_count[slug],
-        controlled_patients_rate: repo.controlled_patients_rate[slug],
-        uncontrolled_patients: repo.uncontrolled_patients_count[slug],
-        uncontrolled_patients_rate: repo.uncontrolled_patients_rate[slug],
+        adjusted_patient_counts: repo.adjusted_patients[slug],
```

Comment:
Given that we've mostly dropped `_patients_count` like how we changed `controlled_patient_counts` to `controlled`, is there any value in keeping `_patients` on `adjusted_patients` and `cumulative_assigned_patients`?

Reply:
Yeah, I can go back and forth on this.  I think the repository does (currently) return things other than just patient related data (like BP measure counts), so it may be a bit early to drop the `patients` suffix.  I do like that idea if we can split up the repository into a more modular interface.